mdoc/Mono.Documentation/exceptions.cs

```diff
@@ -75,7 +75,7 @@ public ExceptionLookup (ExceptionLocations locations)
 				if (member == null)
 					throw new ArgumentNullException ("member");
 
-				var memberDef = member.Resolve ();
+				var memberDef = member.Resolve();
```

Comment:
1. Whitespace-only changes should be avoided.
2. Mono conventions is space before `(`, so this change is wrong, anyway.